packages/core/test/interpreter.test.ts

```diff
@@ -1846,20 +1846,35 @@ describe('interpreter', () => {
 
     expect(spy).not.toHaveBeenCalled();
   });
-});
 
-it(`should execute entry actions when starting the actor after reading its snapshot first`, () => {
-  const spy = jest.fn();
+  it(`should execute entry actions when starting the actor after reading its snapshot first`, () => {
+    const spy = jest.fn();
 
-  const actorRef = interpret(
-    createMachine({
-      entry: spy
-    })
-  );
+    const actorRef = interpret(
+      createMachine({
+        entry: spy
+      })
+    );
 
-  actorRef.getSnapshot();
+    actorRef.getSnapshot();
```

Comment:
To make it more obvious when this is supposed to be called here:
```suggestion
    actorRef.getSnapshot();
    expect(spy).not.toHaveBeenCalled();
```